src/main/scala/cromwell/Main.scala

```diff
@@ -18,19 +18,18 @@ object Actions extends Enumeration {
 }
 
 object Main extends App {
+  val Properties = System.getProperties
+  val LoggerProperty = "CROMWELL_LOGGER"
+  lazy val Log = LoggerFactory.getLogger("main")
```

Comment:
Is that some kind of Scala convention that I wasn't aware of?  Is the convention: any `val` inside of `object` should have an upper-cased first letter of its identifier?